src/compiler/parser.ts

```diff
@@ -9152,6 +9158,16 @@ namespace Parser {
                 return finishNode(factory.createJSDocAugmentsTag(tagName, className, parseTrailingTagComments(start, getNodePos(), margin, indentText)), start);
             }
 
+            function parseSatisfiesTag(start: number, tagName: Identifier, margin: number, indentText: string): JSDocSatisfiesTag {
+                if (some(tags, isJSDocSatisfiesTag)) {
+                    parseErrorAt(tagName.pos, scanner.getTokenPos(), Diagnostics._0_tag_already_specified, tagName.escapedText);
+                }
+
+                const typeExpression = parseJSDocTypeExpression(/*mayOmitBraces*/ true);

```

Comment:
I feel like we *don't* want to allow omitted braces.

Reply:
oke